cli/jhipster.js

```diff
@@ -35,6 +35,10 @@ if (!semver.satisfies(currentNodeVersion, minimumNodeVersion)) {
     process.exit(1);
 }
 
+/* WARNING: due to this code, it's a bit more difficult to test when developing
+ * the upgrade generator, workaround consists in running it this way:
+ * node generator-jhipster/cli/cli.js upgrade
+ */
```

Comment:
should we add a flag to control this behavior like `--use-local` and `--use-global` which can override these